### ISSUE TITLE
riscv64 builds fail with "No usable version of clang found." error

### ISSUE THREAD

**Issue by jkotas** [dotnet/runtime#100215]
## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=615798
Build error leg or test failing: Build / CoreCLR  Product Build linux riscv64 checked / Build CoreCLR Runtime
Pull request: https://github.com/dotnet/runtime/pull/100199

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "No usable version of clang found.",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=615798
**Error message validated:** `[No usable version of clang found.`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 3/25/2024 2:15:49 AM UTC

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[620839](https://dev.azure.com/dnceng-public/public/_build/results?buildId=620839)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/620839/logs/238)||
|[620772](https://dev.azure.com/dnceng-public/public/_build/results?buildId=620772)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/620772/logs/335)|dotnet/runtime#100073|
|[2415977](https://dev.azure.com/dnceng/internal/_build/results?buildId=2415977)|dotnet-runtime|Build product|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2415977/logs/672)|[#38574](https://dev.azure.com/dnceng/internal/_git/dotnet-runtime/pullrequest/38574)|
|[619495](https://dev.azure.com/dnceng-public/public/_build/results?buildId=619495)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/619495/logs/2770)|dotnet/runtime#99696|
|[619641](https://dev.azure.com/dnceng-public/public/_build/results?buildId=619641)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/619641/logs/357)|dotnet/runtime#99849|
|[619427](https://dev.azure.com/dnceng-public/public/_build/results?buildId=619427)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/619427/logs/188)|dotnet/runtime#100303|
|[619204](https://dev.azure.com/dnceng-public/public/_build/results?buildId=619204)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/619204/logs/2624)||
|[618194](https://dev.azure.com/dnceng-public/public/_build/results?buildId=618194)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/618194/logs/251)|dotnet/runtime#100122|
|[618004](https://dev.azure.com/dnceng-public/public/_build/results?buildId=618004)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/618004/logs/328)|dotnet/runtime#99849|
|[617260](https://dev.azure.com/dnceng-public/public/_build/results?buildId=617260)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/617260/logs/452)|dotnet/runtime#99585|
|[617255](https://dev.azure.com/dnceng-public/public/_build/results?buildId=617255)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/617255/logs/483)|dotnet/runtime#99696|
|[617016](https://dev.azure.com/dnceng-public/public/_build/results?buildId=617016)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/617016/logs/238)|dotnet/runtime#100221|
|[616699](https://dev.azure.com/dnceng-public/public/_build/results?buildId=616699)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/616699/logs/338)|dotnet/runtime#100073|
|[611963](https://dev.azure.com/dnceng-public/public/_build/results?buildId=611963)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/611963/logs/1415)|dotnet/runtime#100086|
|[616224](https://dev.azure.com/dnceng-public/public/_build/results?buildId=616224)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/616224/logs/278)|dotnet/runtime#100221|
|[616000](https://dev.azure.com/dnceng-public/public/_build/results?buildId=616000)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/616000/logs/261)|dotnet/runtime#100073|
|[615798](https://dev.azure.com/dnceng-public/public/_build/results?buildId=615798)|dotnet/runtime|Build CoreCLR Runtime|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/615798/logs/227)|dotnet/runtime#100199|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|6|17|17|

**Comment by filipnavara** [comment]
Presumably this is a consequence of https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/976 and https://github.com/dotnet/runtime/pull/100258 should fix it?

**Comment by am11** [comment]
Yup, it's exclusive to release/8.0.

**Comment by jkotas** [comment]
Fixed by #100258
